### PR TITLE
Create oclif commands skeleton in the template - Closes #6075

### LINKED ISSUE #6075: Create oclif commands skeleton in the template
### Description

The template we develop should export `oclif` commands which are available in `lisk-core`. In implementation all the command should be developed in a utility package and then imported to the template. 

### Motivation

This will able user to run those commands e.g. `/run start` with all possible flags, options we use in `lisk-core`. This will also enable use to fix any bug in the utility package to cover up any issue in such commands. User will also be able to just remove the default code and make their own implementation of the command. 

### Acceptance Criteria

- Every command added to template should just be an import and export not a real implementation 
- The test skeleton should also be generated for `account:create`
- All the bootstrap commands to be exported as abstract class

### Additional Information

- For reference we can use the `lisk-core` commands and use `lisk-commander` as library for their reference

### PR DESCRIPTION
**Pull request by ManuGowda**
### What was the problem?

This PR resolves #6075 

### How was it solved?

- Add base commands to commander
- Add commands to `lisk-ts` template

### How was it tested?

1. `cd lisk-sdk`; `yarn; yarn build`
2. `cd commander; yarn link`
3. `mkdir /tmp/test_lisk; cd /tmp/test_lisk`
4. `../lisk-sdk/commander/bin/run init`
5. `yarn link lisk-commander`
6. `./bin/run`
7, `npm run test`

### CODE REVIEW

File: commander/src/bootstrapping/templates/lisk-template-ts/templates/app/src/commands/account/show.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export { AccountShowCommand } from 'lisk-commander';
```
Comment (shuse2) [inline commander/src/bootstrapping/templates/lisk-template-ts/templates/app/src/commands/account/show.ts:1]: For all the commands, it should be either consistently expose as `default` or as object.
ie) some file exposes as `default class` but some as object

File: commander/src/bootstrapping/templates/lisk-template-ts/templates/app/src/commands/start.ts [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+		// eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+		config.plugins[MonitorPlugin.alias] = config.plugins[MonitorPlugin.alias] ?? {};
+		config.plugins[MonitorPlugin.alias].whiteList = flags['monitor-plugin-whitelist']
+			.split(',')
+			.filter(Boolean);
+	}
+};
+
+export class StartCommand extends BaseStartCommand {
+	static flags = {
```
Comment (shuse2) [inline commander/src/bootstrapping/templates/lisk-template-ts/templates/app/src/commands/start.ts:55]: Failing on build here

**Review by shuse2 — CHANGES_REQUESTED** [review]
generally LGTM

File: commander/src/bootstrapping/commands/blockchain/hash.ts [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+	static examples = ['blockchain:hash', 'blockchain:hash --data-path ./data'];
+
+	static flags = {
+		'data-path': flagParser.string({
+			char: 'd',
+			description:
+				'Directory path to specify where node data is stored. Environment variable "LISK_DATA_PATH" can also be used.',
+			env: 'LISK_DATA_PATH',
+		}),
```
Comment (nazarhussain) [inline commander/src/bootstrapping/commands/blockchain/hash.ts:33]: There is a `data-path` in every other command. Would not it be nice to export such flags independently and reuse. Something like below. 

import commonFlags from '../common_flags`

```
static flags = {
     commonFlags.dataPath
}
```

File: commander/src/bootstrapping/commands/start.ts [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+			description:
+				'Seed peers to initially connect to in format of comma separated "ip:port". IP can be DNS name or IPV4 format. Environment variable "LISK_SEED_PEERS" can also be used.',
+		}),
+	};
+
+	// eslint-disable-next-line @typescript-eslint/require-await
+	async run(): Promise<void> {
+		const { flags } = this.parse(StartCommand);
+		const dataPath = flags['data-path'] ? flags['data-path'] : getDefaultPath();
+		this.log(`Starting Lisk Core at ${getFullPath(dataPath)}.`);
```
Comment (nazarhussain) [inline commander/src/bootstrapping/commands/start.ts:114]:
```suggestion
		this.log(`Starting Lisk application at ${getFullPath(dataPath)}.`);
```

**Review by nazarhussain — CHANGES_REQUESTED** [review]
The `dataPath` logic in all the commands is not compatible with custom app. It always point to `~/.lisk/lisk-core` which is wrong in reference to custom app. Hence every command using `dataPath` in one or other way is failing.

**Comment by ManuGowda** [comment]
@nazarhussain i have created a separate issue to address the improvements https://github.com/LiskHQ/lisk-sdk/issues/6087

File: commander/package.json [hunk 5]
```diff
@@ -82,14 +82,17 @@
 		"/docs"
 	],
 	"dependencies": {
+		"lisk-sdk": "5.0.3",
```
Comment (shuse2) [inline commander/package.json:85]: prefer to use each element or framework package from `lisk-commander`

**Review by shuse2 — APPROVED** [review]

**Review by nazarhussain — APPROVED** [review]

**Pull request merged**